### PR TITLE
Fix lambda events

### PR DESCRIPTION
**Pull request by sijis**
This should fix an issue where events may not be added to the correct lambda method.

### CODE REVIEW

File: src/foremast/awslambda/cloudwatch_event/cloudwatch_event.py [hunk 1]
```diff
@@ -82,10 +82,11 @@ def create_cloudwatch_event(app_name, env, region, rules):
...
     json_payload = '{}'.format(json.dumps(json_input))
 
-    target = {"Id": app_name, "Arn": lambda_alias_arn, "Input": json_payload}
+    target = {"Id": app_name, "Arn": lambda_arn, "Input": json_payload}
 
     targets.append(target)
 
-    cloudwatch_client.put_targets(Rule=rule_name, Targets=targets)
+    put_targets_response = cloudwatch_client.put_targets(Rule=rule_name, Targets=targets)
+    LOG.debug(put_targets_response)
```
Comment (e4r7hbug) [inline src/foremast/awslambda/cloudwatch_event/cloudwatch_event.py]: Add a descriptive string to this DEBUG.

File: src/foremast/utils/awslambda.py [hunk 2]
```diff
@@ -111,7 +112,8 @@ def add_lambda_permissions(function='',
     try:
         lambda_client.add_permission(**add_permissions_kwargs)
         response_action = 'Add permission with Sid: {}'.format(statement_id)
-    except boto3.exceptions.botocore.exceptions.ClientError:
+    except boto3.exceptions.botocore.exceptions.ClientError as error:
+        LOG.debug(error)
```
Comment (e4r7hbug) [inline src/foremast/utils/awslambda.py]: Also add a descriptive string to this DEBUG.

File: src/foremast/awslambda/cloudwatch_event/cloudwatch_event.py (hunk 1 above)
Comment (sijis) [inline src/foremast/awslambda/cloudwatch_event/cloudwatch_event.py]: Added descriptive message.

File: src/foremast/utils/awslambda.py (hunk 2 above)
Comment (sijis) [inline src/foremast/utils/awslambda.py]: Added descriptive message.

**Pull request merged**